myuser/myuser.go

```diff
@@ -0,0 +1,16 @@
+package myuser
+
+import (
+	"os/user"
+)
+
+// GetUserHomeDir gets path to home directory.
+func GetUserHomeDir() (string, error) {
+	user, userError := user.Current()
```

Comment:
Indentation :)

Reply:
I've checked this and I still have the same indentation.